charts/consul/templates/crd-gatewayclassconfigs.yaml

```diff
@@ -107,6 +107,10 @@ spec:
                                 type: string
                             type: object
                         type: object
+                      portModifier:
+                        description: PortModifier specifies the value to be added
+                          to every port value for listeners on this gateway
```

Comment:
I think this requires a regen to include the added bit about avoiding privileged ports